app/helpers/constants.rb

```diff
@@ -12,4 +9,5 @@ module Constants
 
   FOOTER_NUMBER = '(555)-555-5555'.freeze
   FOOTER_EMAIL = 'support@bookstore.com'.freeze
+  REVIEW_DATE_MASK = '%m/%d/%Y'.freeze
```

Comment:
I think it's not only for review
```suggestion
  DATE_MASK = '%m/%d/%Y'.freeze
```